browser_extension/css/font-awesome.css

```diff
@@ -0,0 +1,2199 @@
+/*!
```

Comment:
We already have a font awesome library [/browser_extension/js/fontawesome/fontawesome-all.js](https://github.com/marcobeltempo/chaddon/blob/dev/browser_extension/js/fontawesome/fontawesome-all.js)

To use the existing one, place the following tag just before the closing `</body>` tag.
```html
  <script type="text/javascript" src="/js/fontawesome/fontawesome-all.js"></script>
```